1. Start with coding task

In [5]:
def jprobability(pA, pB):
  return pA * pB

def mprobability(pA, pB):
  return pA + pB - jprobability(pA, pB)

def cprobability(pBA, pA, pB):
  return (pBA * pA) / pB

pA = 0.3
pB = 0.4
pBA = 0.8

print(f"Joint probability: {jprobability(pA, pB)}")
print(f"Marginal probability {mprobability(pA, pB)}")
print(f"Conditional probability: {cprobability(pBA, pA, pB)}")

Joint probability: 0.12
Marginal probability 0.58
Conditional probability: 0.6


2. Interactive question.

The joing probability of A and B is **0.12** assuming they are independent

3. Feedback


The joint probability of two independent events is the product of their individual probabilities.